packages/api-v4/src/account/types.ts

```diff
@@ -391,8 +394,10 @@ export const EventActionKeys = [
   'linode_migrate_datacenter_create',
   'linode_migrate_datacenter',
   'linode_migrate',
+  'linode_migration',
   'linode_mutate_create',
   'linode_mutate',
+  'linode_power_on_off',
```

Comment:
```suggestion
  'linode_migrate',
  'linode_mutate_create',
  'linode_mutate',
  'linode_poweroff_on',
```

Event actions as mentioned in the APISpec, or am I missing something?

Reply:
I think you're correct, I will verify with the API development team.

Reply:
`linode_migrate` is an existing action type, so I was a little confused. I left a comment on the API spec for further clarification.